pom.xml

```diff
@@ -278,7 +278,7 @@
 			<dependency>
 				<groupId>org.liquibase</groupId>
 				<artifactId>liquibase-core</artifactId>
-				<version>2.0.5</version>
+				<version>3.6.3</version>
```

Comment:
Does the ticket also require this upgrade? You pushed to the same base branch. You can follow this https://wiki.openmrs.org/display/docs/Pull+Request+Tips?

Reply:
> Does the ticket also require this upgrade? You pushed to the same base branch. You can follow this https://wiki.openmrs.org/display/docs/Using+Gits

but it's like this page you are linking me to doesn't exist

Reply:
It's this https://wiki.openmrs.org/display/docs/Using+Git that's a wrong link